edi_account_oca/views/res_partner.xml

```diff
@@ -7,7 +7,7 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="account.view_partner_property_form" />
         <field name="arch" type="xml">
-            <!-- TODO: Move this conf inside `edi` page from edi_oca. -->
+            <!-- TODO: Move this conf inside `edi` page from edi_core_oca. -->
             <group name="accounting_entries" position="after">
```

Comment:
I'd drop this change in this PR and move it to its own. The page is already available on the partner.